### PR TITLE
Feature/ADF-525/Improve ACL logs

### PR DESCRIPTION
**Pull request by shpran**
# [ADF-525](https://oat-sa.atlassian.net/browse/ADF-525)

## Changes
* Added new log information
* Updated old log information
* Small code refactoring

## Required PRs
* https://github.com/oat-sa/generis/pull/943

## Companion PRs
* https://github.com/oat-sa/tao-core/pull/3247

## Tasks
- [ ] Update `oat-sa/generis` dependency

### CODE REVIEW

File: models/FuncAcl.php [hunk 1]
```diff
@@ -15,73 +15,89 @@
...
-            
+
             switch (count($elements)) {
                 case 1:
                     $extension = reset($elements);
                     $accessService->grantExtensionAccess($rule->getRole(), $extension);
                     break;
                 case 2:
-                    list($extension, $shortName) = $elements;
+                    [$extension, $shortName] = $elements;
```
Comment (hectoras) [inline models/FuncAcl.php:96]: Suggestion: [Array destructuring](https://www.php.net/manual/en/migration71.new-features.php#migration71.new-features.symmetric-array-destructuring) was introduced in PHP 7.1, maybe we should add require directive for `"php": ">=7.1"` in the [composer.json file for this extension](https://github.com/oat-sa/extension-tao-funcacl/blob/feature/adf-525/improve-acl-logs/composer.json).

(By the way, if we don't add that, this depends on tao-core v47, and it [already declares a dependency on 7.1](https://github.com/oat-sa/tao-core/blob/v47.0.0/composer.json))

**Review by hectoras — APPROVED** [review]
Left a couple of suggestions, but LGTM overall :rocket:

File: models/FuncAcl.php (hunk 1 above)
Comment (shpran) [reply models/FuncAcl.php:96]: Don't worry about it 🙂 
All of our customers and all of our environments use PHP 7.2

**Review by gabrielfs7 — APPROVED** [review]
Good job! 

- [ ] **Please, do not forget to update composer.json with new tao-core/generis version.**

...

- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Comment by AntonNeviarouski** [comment]
QA verified

**Pull request merged**